docs/paper/admin/reference/configuration/commands-configuration.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: /reference/commands-configuration
+---
+
+# Commands Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for `commands.yml`.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config data={`
+command-block-overrides:
+  default: '[<command to override>]'
+  description: 'What commands to use the vanilla versions in command blocks, instead of bukkit or plugin provided versions.
```

Comment:
Worded weirdly; More like “Which commands should override…” also is it overriding vanilla or overriding bukkit? If it is the Vanilla command being used i think override is a weird word in the description (i know it’s in the key) just because it’s more of a defaulting action